duckduckgo/iOS
# Add Events Firing for Phishing Detection Settings: Point to BSK

Task/Issue URL: https://app.asana.com/0/72649045549333/1208270234071171/f
Tech Design URL: N/A

**Description**:
In [✓ Phishing Detection Pixels](https://app.asana.com/0/0/1207699541075655/f) we defined several pixels to include, and one of them did not make it into the initial release: 
- macos.phishing-detection.feature-toggled:
  - Parameters:
    - newState - on/off, indicating whether the user turned it on or off
- Condition
  - triggers when a user disables the feature from the settings page
- Reasoning
  - the aim is to use this as a "do-no-harm" KPI, to determine how many users are actively choosing to disable/enable phishing protection



**Steps to test this PR**:
1. No testing required on iOS, this is just a library change introduction in BSK, used by macOS.
2. Ensure CI goes green



**Definition of Done (Internal Only)**:


**Copy Testing**:


**Orientation Testing**:


**Device Testing**:


**OS Testing**:


**Theme Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
